### PR TITLE
Feature/KITOSUDV-3547

### PR DESCRIPTION
**Pull request by a-naskret**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [ ] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [ ] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [ ] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [ ] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Presentation.Web/Models/API/V1/ItSystemUsage/GDPR/GDPRExportReportColumns.cs [hunk 1]
```diff
@@ -16,6 +16,7 @@ public class GDPRExportReportColumns
         public static CsvColumnIdentity DataProcessorControl = new CsvColumnIdentity(nameof(DataProcessorControl), "Tilsyn/kontrol af databehandleren");
         public static CsvColumnIdentity LinkToDirectory = new CsvColumnIdentity(nameof(LinkToDirectory), "Link til fortegnelse");
         public static CsvColumnIdentity RiskAssessment = new CsvColumnIdentity(nameof(RiskAssessment), "Foretaget risikovurdering");
+        public static CsvColumnIdentity RiskAssessmentDate = new CsvColumnIdentity(nameof(RiskAssessmentDate), "RiskAssessmentDate <- change the text");
```
Comment (a-naskret) [inline Presentation.Web/Models/API/V1/ItSystemUsage/GDPR/GDPRExportReportColumns.cs:19]: change text

File: Presentation.Web/app/components/it-system/it-system-overview.controller.ts [hunk 2]
```diff
@@ -812,7 +812,16 @@
                         .withId("note")
                         .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.Contains)
                         .withContentOverflow()
-                        .withSourceValueEchoRendering());
+                        .withSourceValueEchoRendering())
+                .withColumn(builder =>
+                    builder
+                        .withDataSourceName("RiskAssessmentDate")
+                        .withTitle("Risk assessment date <- change the text")
```
Comment (a-naskret) [inline Presentation.Web/app/components/it-system/it-system-overview.controller.ts:819]: change text

File: Tests.Integration.Presentation.Web/ItSystem/ItSystemUsageGDPRTest.cs [hunk 3]
```diff
@@ -471,6 +494,8 @@ public class GdprExportReportCsvFormat
         public string DirectoryUrl { get; set; }
         [Name("Foretaget risikovurdering")]
         public string RiskAssessment { get; set; }
+        [Name("RiskAssessmentDate <- change the text")]
```
Comment (a-naskret) [inline Tests.Integration.Presentation.Web/ItSystem/ItSystemUsageGDPRTest.cs:497]: change text

File: Infrastructure.DataAccess/Migrations/202212070659101_AddedRiskAssessmentDateToReadModel.cs [hunk 4]
```diff
@@ -0,0 +1,18 @@
+﻿namespace Infrastructure.DataAccess.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class AddedRiskAssessmentDateToReadModel : DbMigration
+    {
+        public override void Up()
+        {
+            AddColumn("dbo.ItSystemUsageOverviewReadModels", "RiskAssessmentDate", c => c.DateTime(precision: 7, storeType: "datetime2"));
```
Comment (mrjsawdk) [inline Infrastructure.DataAccess/Migrations/202212070659101_AddedRiskAssessmentDateToReadModel.cs:10]: remember that this is a filterable field, so we need an index

File: Presentation.Web/Models/API/V1/ItSystemUsage/GDPR/GDPRExportReportColumns.cs (hunk 1 above)
Comment (mrjsawdk) [reply Presentation.Web/Models/API/V1/ItSystemUsage/GDPR/GDPRExportReportColumns.cs:19]: why not use the one from the UI? 
image

File: Presentation.Web/Models/API/V1/ItSystemUsage/GDPR/GDPRExportReportCsvMapper.cs [hunk 5]
```diff
@@ -59,6 +60,11 @@ private static string MapBoolean(bool input)
             return input ? Yes : No;
         }
 
+        private static string MapDateTime(DateTime? input)
+        {
+            return input == null ? "" : input.GetValueOrDefault().ToShortDateString();
```
Comment (mrjsawdk) [inline Presentation.Web/Models/API/V1/ItSystemUsage/GDPR/GDPRExportReportCsvMapper.cs:65]: does this render like the other fields in kitos: date-month-year

File: Presentation.Web/app/components/it-system/it-system-overview.controller.ts (hunk 2 above)
Comment (mrjsawdk) [reply Presentation.Web/app/components/it-system/it-system-overview.controller.ts:819]: why not use the UI text 
image

File: Presentation.Web/app/components/it-system/it-system-overview.controller.ts [hunk 6]
```diff
@@ -812,7 +812,16 @@
...
+                        .withId("LatestRiskAssessmentDate")
+                        .withDataSourceType(Utility.KendoGrid.KendoGridColumnDataSourceType.Date)
+                        .withStandardWidth(170)
+                        .withRendering(dataItem => Helpers.RenderFieldsHelper.renderDate(dataItem.RiskAssessmentDate))
+                        .withExcelOutput(dataItem => Helpers.ExcelExportHelper.renderDate(dataItem.RiskAssessmentDate)));
```
Comment (mrjsawdk) [inline Presentation.Web/app/components/it-system/it-system-overview.controller.ts:824]: not following your own learnings here... no need to repeat the same rendering if the UI rendering is OK for excel output

File: Presentation.Web/Models/API/V1/ItSystemUsage/GDPR/GDPRExportReportCsvMapper.cs (hunk 5 above)
Comment (mrjsawdk) [reply Presentation.Web/Models/API/V1/ItSystemUsage/GDPR/GDPRExportReportCsvMapper.cs:65]: no, 
image

Comment (mrjsawdk) [reply Presentation.Web/Models/API/V1/ItSystemUsage/GDPR/GDPRExportReportCsvMapper.cs:65]: in order to make sure, you should specify the format.
The one above works on the right server locale, but specifying the format will allow you to **guarantee** the format

**Review by mrjsawdk — CHANGES_REQUESTED** [review]
@a-naskret A few adjustments

**Review by mrjsawdk — APPROVED** [review]
@a-naskret merge when green

**Pull request merged**